### ISSUE TITLE
Apply classes on click and current date

### ISSUE THREAD

**Issue by ManiAkasapu** [bradberger/angular-material-calendar#13]
This is more of feature request.

I see there is no way to Add a class to a date.
For Example, Instead of injecting HTML It is easy to change the background color (By applying the class) in case of an event.

**Comment by bradberger** [comment]
You're right, I don't think that's possible right now. I can see how that would be useful, and think it's worth checking out how we could implement it. I don't think I'm going to get to it right away, though, so let's leave the issue open in hopes someone else might be able to implement it.
